Competition2017/src/competition/subsystems/collector/commands/StopCollectorCommand.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.collector.commands;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.collector.CollectorSubsystem;
+import xbot.common.command.BaseCommand;
+
+
+public class StopCollectorCommand extends BaseCommand {
+    final CollectorSubsystem collectorSystem;
+    private static Logger log = Logger.getLogger(StopCollectorCommand.class);
+    
+    @Inject
+    public StopCollectorCommand(CollectorSubsystem collectorSystem){
+        this.collectorSystem = collectorSystem;   
+        this.requires(collectorSystem);
+    }
+    @Override
+    public void initialize() {
+        log.info("Stopping Collector");
```

Comment:
This log line should match the others in related commands. I'd prefer "Initializing <someName>Command" or "Starting <someName>Command".